### ISSUE TITLE
[studio-ui] Add a UI for bulk publish all

### ISSUE THREAD

**Issue by avasquez614** [craftercms/craftercms#5682]
### Duplicates

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

Bulk publish all API was added in this ticket: https://github.com/craftercms/craftercms/issues/5591. We need a UI to do a bulk publish all.

### Describe the solution you'd like

Add a simple UI screen to bulk publish all, or modify the current bulk publish screen to add a button that executes a bulk publish all.

**Comment by rart** [comment]
@sumerjabri @avasquez614 I'm just being made aware of this [API](https://github.com/craftercms/craftercms/issues/5591). Reviewing, there's no submission comment and the publishing target is optional.
- Shouldn't there be an optional submission comment?
- Shouldn't the publishing target be mandatory?

@jvega190 update the Publish On Demand widget to accommodate the new option.

![image](https://user-images.githubusercontent.com/3928341/179208190-2af610da-9a11-4c53-ae17-ab0595a8cbae.png)

**Comment by sumerjabri** [comment]
@rart yes, I think you're right and this should be fixed. CC @avasquez614 

@jmendeza please begin fixing the API to accept an optional submission comment and a mandatory publishing target (env).

**Comment by jmendeza** [comment]
Added `submissionComment` (optional) parameter, added "site exists" check, made `publishingTarget` required

**Comment by sumerjabri** [comment]
@alhambrav please check with @rart on what needs to be updated in the docs.
